### PR TITLE
Working RAW-Graphs

### PR DESCRIPTION
**Pull request by odeimaiz**
- [x] Links from Preferences replaced by LinkButtons
- [x] Reverse Proxy fixed to make RAW-Graphs work
- [x] RawGraphs errors transmitted to front-end

### CODE REVIEW

File: services/web/client/source/class/qxapp/component/widget/LinkButton.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+     2019 IT'IS Foundation, https://itis.swiss
+
+   License:
+     MIT: https://opensource.org/licenses/MIT
+
+   Authors:
+     * Odei Maiz (odeimaiz)
+
+************************************************************************ */
+
```
Comment (oetiker) [inline services/web/client/source/class/qxapp/component/widget/LinkButton.js]:
```suggestion
/**
 * A button with low optical impact presenting as a simple weblink
 *
 * *Example*
 *
 * Here is a little example of how to use the widget.
 *
 * <pre class='javascript'>
 *   var link = new qxapp.component.widget.LinkButton(this.tr("oSparc"),"https://osparc.io");
 *   this.getRoot().add(link);
 * </pre>
 */
```

File: services/web/client/source/class/qxapp/component/widget/LinkButton.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+qx.Class.define("qxapp.component.widget.LinkButton", {
+  extend: qx.ui.form.Button,
+
```
Comment (oetiker) [inline services/web/client/source/class/qxapp/component/widget/LinkButton.js]:
```suggestion
/**
 * @param label {String} Label to use
 * @param url {String} Url to point to
 * @param height {Integer?12} Height of the link icon
 */
```

File: services/web/client/source/class/qxapp/component/widget/LinkButton.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    this.base(arguments, label);
+
+    this.set({
+      icon: "@FontAwesome5Solid/external-link-alt/"+height,
+      iconPosition: "right",
+      allowGrowX: false
+    });
+
+    this.addListener("execute", () => {
+      window.open(url);
```
Comment (oetiker) [inline services/web/client/source/class/qxapp/component/widget/LinkButton.js]: does this open in a new window automatically?

Comment (odeimaiz) [reply services/web/client/source/class/qxapp/component/widget/LinkButton.js]: Yep

File: services/web/client/source/class/qxapp/component/widget/LinkButton.js (hunk 1 above)
Comment (odeimaiz) [reply services/web/client/source/class/qxapp/component/widget/LinkButton.js]: Doc: Class description

**Pull request merged**